src/main/kotlin/net/theevilreaper/dartpoet/code/writer/ConstructorWriter.kt

```diff
@@ -39,17 +41,17 @@ class ConstructorWriter {
             if (spec.parameters.isNotEmpty()) {
                 writer.emit(",$NEW_LINE")
             }
-            writer.emit("{")
-            writer.emit("\n")
-            writer.indent(1)
+            writer.emit("$CURLY_OPEN")
+            writer.emit(NEW_LINE)
+            writer.indent()
 
             spec.requiredAndNamedParameters.emitParameters(writer, emitBrackets = false, emitSpace = false, forceNewLines = true) {
                 it.write(writer)
             }
 
-            writer.unindent(1)
+            writer.unindent()
             writer.emit(NEW_LINE)
-            writer.emit("}")
+            writer.emit("$CURLY_CLOSE")
```

Comment:
Its really required to put it in ""

Reply:
At the moment yes because the CURLY_CLOSE constant is a char and not a string